x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/card_selector_list.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useEffect } from 'react';
+import { EuiPanel, EuiFlexGroup, EuiFlexItem, EuiTitle, EuiText, EuiSpacer } from '@elastic/eui';
+import type { RulesCardItemId } from '../rules/types';
+import type { AlertsCardItemId } from '../alerts/types';
+import type { DashboardsCardItemId } from '../dashboards/types';
+import { useCardSelectorListStyles } from './card_selector_list.styles';
+
+const INITIAL_SCROLL_ANIMATION_DURATION = 500;
+const SCROLL_ANIMATION_DURATION = 300;
+
+export enum CardSelectorListItemAssetType {
+  video = 'video',
+  image = 'image',
+}
+export interface CardSelectorListItem {
+  id: RulesCardItemId | AlertsCardItemId | DashboardsCardItemId;
+  title: string;
+  description: string;
+  asset: {
+    type: CardSelectorListItemAssetType;
+    source: string;
+    alt: string;
+  };
```

Comment:
The `asset` has nothing to do with the `CardSelectorList`, it is used only by `OnboardingCardContentAsset` upstream. Can we remove it from this type?